docs/DeveloperGuide.md

```diff
@@ -300,7 +300,7 @@ The rationale for overloading such methods are given below:<br>
 
 Timetable handles adding, deleting and getting all instances of stored events in a given time period. All scheduling, retrieving and processing of events are done here
 
-The timetable component stores an array of events. On launch, an empty timetable will be created. All stored events will be loaded via the StorageManger. 
+The timetable component stores five arrays of events. On launch, an empty timetable will be created. All stored events will be loaded via the StorageManger. 
```

Comment:
Maybe can state what the five arrays are? I believe you will elaborate on them later in the doc right? So just list out the five?